### DISCUSSION TITLE
Why is nothing below the baseline shown after exporting the font?

### DISCUSSION THREAD

**Discussion by jilotta** [fontforge/fontforge#5285] [Q&A]
I am creating a font. Several letters have parts that are below the baseline, such as Ц, Щ and a comma. When I export the font to truetype or opentype, eveything but the parts that are below the baseline.

How Ц is supposed to look like: https://i.ibb.co/ZcBLKtg/image.png
How it looks like afer exporting: https://i.ibb.co/B4HN8vC/image.png

What did i do wrong?

**Comment by jilotta** [comment]
bump

**Comment by Bry10022** [comment]
I think this might be because of the WinDescent value. Try adjusting it as anything outside the WinAscent and WinDescent gets clipped in Windows.
